dll/win32/shell32/wine/shellpath.c

```diff
@@ -146,90 +146,136 @@ static BOOL WINAPI PathMakeAbsoluteW(LPWSTR path)
 }
 #endif
 
-/* NOTE: GetShortPathName fails if the pathname didn't exist.
-         GetShortPathNameAbsentW should set the short path name that even doesn't exist. */
-static DWORD GetShortPathNameAbsentW(LPCWSTR pszLong, LPWSTR pszShort, DWORD cchShort)
-{
-    FIXME("GetShortPathNameAbsentW(%ls, %p, %ld): stub\n", pszLong, pszShort, cchShort);
-    StringCchCopyW(pszShort, cchShort, pszLong);
-    return lstrlenW(pszShort);
-}
-
-BOOL WINAPI IsLFNDriveW(LPCWSTR lpszPath);
-
-/* @unconfirmed */
+/* @implemented */
```

Comment:
Maybe add SAL2 annotation here instead of just `@implemented`. Also it looks like this is Wine code (i.e. `#ifdef __REACTOS__` is needed).

Reply:
```txt
dll/win32/shdocvw             # Synced to WineStaging-4.18
dll/win32/shell32             # Forked at Wine-20071011
dll/win32/shfolder            # Synced to WineStaging-3.3
dll/win32/shlwapi             # Synced to WineStaging-3.3
```

`SHELL32` is already forked.

Reply:
Ah ok, but interesting that there are some existing `__REACTOS__` guards in this file :slightly_smiling_face:

Reply:
Dunno what's the deal with this `@implemented` tbh.

Reply:
Deleted.